### DISCUSSION TITLE
Creating a headmap of the number of files modified in the vault per day

### DISCUSSION THREAD

**Discussion by jadoff630** [Richardsl/heatmap-calendar-obsidian#41] [Q&A]
I was wondering if it was possible to create a heatmap of the number of files modified per day in the vault as opposed to things based on a YAML attribute. Since dataview can pull all the files modified in a day, and you can also just list the number with dataviewjs, I figured it should be possible to use that info in the heatmap, but can't figure out how that would work. 

Thanks!

**Comment by xinbenlv** [comment]
I think it's a good idea. The first thing to ask is how do you detect "modify". You'd probably have to have some history. For example I use git for my Obsidian vault so I can lookup the `git history`. But such information might not exist if you use other sync service or are not using any sync service to track file history.
